### ISSUE TITLE
how to train

### ISSUE THREAD

**Issue by KDgggg** [rohit901/cooperative-foundational-models#4]
hello, i am wodering how can i train the model on my own datasets

**Comment by rohit901** [comment]
Hello,

thank you for your interest in our work. Our work is a training-free approach which utilizes pre-trained foundational models to complement Mask-RCNN. 

To use your custom dataset, I would recommend you to read the guide for detectron2 on using custom dataset (https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html)
You may also refer to other docs on the above website for some insights and help on how to change the data loader used in the script. 

Further, the actual data loader in the main script is in this line: https://github.com/rohit901/cooperative-foundational-models/blob/fb45630c46d75f7a1856654665191aeddeb3eb98/scripts/novel_object_detection/main.py#L69

**Comment by KDgggg** [comment]
> Hello,
> 
> thank you for your interest in our work. Our work is a training-free approach which utilizes pre-trained foundational models to complement Mask-RCNN.
> 
> To use your custom dataset, I would recommend you to read the guide for detectron2 on using custom dataset (https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html) You may also refer to other docs on the above website for some insights and help on how to change the data loader used in the script.
> 
> Further, the actual data loader in the main script is in this line:
> 
> https://github.com/rohit901/cooperative-foundational-models/blob/fb45630c46d75f7a1856654665191aeddeb3eb98/scripts/novel_object_detection/main.py#L69

may i finetune on my custom dataset in this model

**Comment by rohit901** [comment]
You can use our approach on your custom dataset, you can refer to the linked docs above, and also read the paper once. Our approach does not require any training, and its a training-free method.
